app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/mandatory.hbs

```diff
@@ -0,0 +1,2 @@
+{{page-title "Mandatory"}}
```

Comment:
I'm not sure if this is needed, but if it is, it needs to be translated :smile:

Reply:
It's better to specify, in dutch, don't you think?

Reply:
I just think that it might make more sense to only show a `Bewerk mandataris` breadcrumb but hard to say.